include/grpc/impl/codegen/compression_types.h

```diff
@@ -101,15 +100,15 @@ typedef struct grpc_compression_options {
    * precedence over \a default_algorithm.
    * TODO(dgq): currently only available for server channels. */
   struct {
-    bool is_set;
+    int is_set;
```

Comment:
I think it did... for several releases... so I *think* we're ok (though the C++ stuff fails with it, so worth fixing)